debian/neo-go.docs

```diff
@@ -0,0 +1,4 @@
+docs/*
+CONTRIBUTING.md
+ROADMAP.md
+README.md
```

Comment:
Add `pkg/rpcclient/doc.go` (and `pkg/rpcclient/doc_test` to examples section)? An important one.

Reply:
What's packaged here is the neo-go binary that implements a node, a compiler and all of these things. `rpcclient` is a programmer-facing API, it's somewhat different and should not be packaged as a document (it's Go code after all).